build.gradle

```diff
@@ -26,9 +25,48 @@ plugins {
     id 'net.researchgate.release' version '2.3.5'
 }
 
+subprojects {
+
+    group = "com.containersol.minimesos"
+    version = rootProject.version.toString()
+
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://jitpack.io"
```

Comment:
I would be a little bit careful with putting jitpack at the very top level. _Potentially_ someone from the outside can screw up everything if they just manage to amend a single dependency. Which isn't too far fetched IMO.